packages/domain/lib/src/models/movie_model/movie/movie.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:domain/src/models/movie_model/movie/ids.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'movie.g.dart';
+
+const imageUrl = 'http://img.omdbapi.com/?apikey=5e582eee&i=';
+
+@JsonSerializable()
+class Movie {
+  final String? title;
+  final int? year;
+  final Ids? ids;
+  final String? tagline;
+  final String? overview;
+  final String? released;
+  final int? runtime;
+  final String? country;
+  final String? trailer;
+  final String? homepage;
+  final String? status;
+  final double? rating;
+  final int? votes;
+  @JsonKey(name: 'comment_count')
+  final int? commentCount;
+  @JsonKey(name: 'updated_at')
+  final String? updatedAt;
+  final String? language;
+  @JsonKey(name: 'available_translations')
+  final List<String>? availableTranslations;
+  final List<String>? genres;
+  final String? certification;
+  final String image;
+  final int stars;
+
+  Movie(
```

Comment:
I recommend using "@JsonSerializable(fieldRename: FielRename.snake)" for speed, so as not to write "@JsonKet(name: ...)" all the time

Reply:
o_O cool feature, thx!